modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/details.jsp

```diff
@@ -83,9 +83,46 @@ if (organization != null) {
 <liferay-ui:error exception="<%= NoSuchCountryException.class %>" message="please-select-a-country" />
 
 <div class="<%= OrganizationLocalServiceUtil.isCountryEnabled(type) ? StringPool.BLANK : "hide" %>" id="<portlet:namespace />countryDiv">
-	<aui:select label="country" name="countryId" />
+	<aui:select label="country" name="countryId" required="<%= true %>">
```

Comment:
are these displayed correctly? Because before there were just the two field, but now we have the two fields + two labels. Are the label placed correctly?

Reply:
label is only for region. Rest works correctly.